CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres
 to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.3.0][1.1.3.0]
```

Comment:
question: Based on the contained changes so far, wouldn't it be version "1.1.2.6" instead?

Reply:
You are right.
I changed it in the Jira release.